_layouts/trademarks.html

```diff
@@ -0,0 +1,88 @@
+                This page summarizes the key points of the policy, but please note that
+                the official policy always takes precedence.</p>
+
+                <ul>
+                  <li><strong>Software products,</strong> whether commercial or open source, are not allowed to use “OpenWhisk” in their name, except in the form “powered by Apache OpenWhisk" or “for Apache OpenWhisk” when following
+                <a href="https://www.apache.org/foundation/marks/faq/#products">these specific
+                guidelines</a>.
+                    <ul>
+                      <li>Names derived from “OpenWhisk” are also not allowed.</li>
+                      <li>Company names may not include “OpenWhisk”.</li>
+                      <li>Package identifiers (e.g., Maven coordinates) may include
+                       “openwhisk”, but the full name used for the software package should
+                       follow the naming policy above.</li>
+                    </ul>
+                  </li>
+                  <li>
+                    <p><strong>Written materials</strong> must refer to the project as “Apache OpenWhisk” in the first and most prominent mentions. Materials from software vendors or software-related service providers must follow
+                <a href="https://www.apache.org/foundation/marks/guide#howoften">stricter
+                guidelines</a>, including using the full project name “Apache OpenWhisk" in more
+                locations, and proper
+                <a href="https://www.apache.org/foundation/marks/faq/#attribution">trademark
+                attribution</a> on every page.</p>
+                  </li>
+                  <li>
+                    <p><strong>Logos</strong> derived from the OpenWhisk logo are not allowed.</p>
+                  </li>
+                  <li>
+                    <p><strong>Domain names</strong> containing “openwhisk” are not permitted without written permission from the Apache OpenWhisk PMC. To request permission, email
+                <a href="mailto:private@openwhisk.apache.org">private@openwhisk.apache.org</a> and CC
+                <a href="mailto:trademarks@apache.org">trademarks@apache.org</a>.
+                (<a href="https://www.apache.org/foundation/marks/domains.html">Details</a>)</p>
+                  </li>
+                  <li><strong>Events, books, and merchandise</strong> have their own guidelines.
+                Please refer to the
+                <a href="https://www.apache.org/foundation/marks/">ASF trademark policy</a>
+                and <a href="https://www.apache.org/foundation/marks/faq/">FAQ</a> for details.</li>
+                </ul>
+
+                <p>We appreciate the community’s help in respecting these guidelines.
+                If you have any questions about the Apache trademark policy, please email
+                <a href="mailto:trademarks@apache.org">trademarks@apache.org</a>.</p>
```

Comment:
I would say "please ask on the OpenWhisk dev list or email  <a href="mailto:trademarks@apache.org">trademarks@apache.org</a>"